src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -146,6 +146,9 @@ void goto_analyzer_parse_optionst::get_command_line_options(optionst &options)
     options.set_option("error-label", cmdline.get_values("error-label"));
 #endif
 
+  // all checks supported by goto_check
+  PARSE_OPTIONS_GOTO_CHECK(cmdline, options);
```

Comment:
Ah! I had noticed at some point that I had to use `goto-instrument --*-check` before using `goto-analyzer`. Debugging `goto-analyzer` is still on my backlog...

Reply:
Yeah, a historical fuck up, esp. as the `GOTO_CHECK` options where given in `--help` and the options were parsed...